client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -455,7 +455,7 @@ function PurchaseMetaExpiration( {
 	const translate = useTranslate();
 	const moment = useLocalizedMoment();
 	const isProductOwner = purchase?.userId === useSelector( getCurrentUserId );
-	const isAutorenewalEnabled = purchase ? ! isExpiring( purchase ) : null;
+	const isAutorenewalEnabled = purchase?.isAutoRenewEnabled ?? false;
```

Comment:
This change in logic results in a downstream issue for the case where `isAutorenewalEnabled` is true and we don't actually have a renew date.

<img width="534" alt="Screenshot 2022-07-21 at 10 09 04" src="https://user-images.githubusercontent.com/3376401/180163523-1a8fb5af-2586-49ee-8916-a689a47e19b1.png">

I tripped over this because I had a subscription with autorenew enabled but no payment method. (I was able to explicitly remove the original payment method via `/me/purchases/payment-methods`.)

I think that we may need an additional check on L472 to make sure we have a `renewDate` present, but it may be better to check whether the purchase has a linked payment method, as that is the reason we aren't going to attempt the renewal.

Reply:
By updating this check on L472

```js
const subsBillingText =	isAutorenewalEnabled && ! hideAutoRenew && hasPaymentMethod( purchase )
```

It will not appear like this
![image](https://user-images.githubusercontent.com/6851384/180177481-66e5fcaa-7241-40c7-8463-d7cec8e1c3cb.png)

Let me know if you meant something different.